### DISCUSSION TITLE
Retirement of Office 365 connectors within Microsoft Teams

### DISCUSSION THREAD

**Discussion by francesco-quaratino** [dagster-io/dagster#24739] [Community]
Am I right saying the current implementation of [dagster_msteams/card.py](https://github.com/dagster-io/dagster/blob/master/python_modules/libraries/dagster-msteams/dagster_msteams/card.py) is not ready for the [Retirement of Office 365 connectors within Microsoft Teams](https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/)?

As my understanding, with the retirement of Incoming Webhooks, and the introduction of Workflows, the content type of the card payload changes, starting from the content_type from "application/vnd.microsoft.card.hero" to "application/vnd.microsoft.card.adaptive".

Please can anyone shed some lights on it for me, thanks.

**Comment by garethbrickman** [comment]
Issue for tracking this: https://github.com/dagster-io/dagster/issues/23223
